src/components/Card/index.tsx

```diff
@@ -342,6 +360,15 @@ const Card: React.FC<CardPropTypes> = ({
           {leftAdornment && <LeftAdornment>{leftAdornment}</LeftAdornment>}
 
           <TextContent hasImage={hasImage}>
+            {header && (
+              <Header>
+                {header.icon}
+                {header.text && (
+                  <HeaderText color={header.color}>{header.text}</HeaderText>
+                )}
+              </Header>
+            )}
```

Comment:
This is still a bit inconsistent with the footer, the right adornment, etc. 

I think we should have the same implementation as the footer prop, and then you can pass JSX to that prop in order to render for example the icon and the text.

Reply:
I thought this would be a bit repetitive in Sirius to do as there will be quite some places where we need to add that period section with icon and text but indeed, this is very general so everybody can use it as they like. Updated!